pkg/profile/profile.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var DevPreviewEnv string = "RHOAS_DEV"
+var DevPreviewEnv = "RHOAS_DEV"
```

Comment:
This could actually be a a `const` while at it.

Reply:
```suggestion
const DevPreviewEnv = "RHOAS_DEV"
```